web/controllers/stripe_customer_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule CodeCorps.StripeCustomerController do
+  use CodeCorps.Web, :controller
+  use JaResource
+
+  alias CodeCorps.StripeCustomer
+
+  plug :load_and_authorize_resource, model: StripeCustomer, only: [:show]
+  plug :load_and_authorize_changeset, model: StripeCustomer, only: [:create]
+  plug JaResource
+
+  def handle_create(conn, attributes) do
+    attributes
+    |> CodeCorps.StripeService.create_customer
+    |> handle_stripe_response(attributes, conn)
+  end
+
+  defp handle_stripe_response({:ok, stripe_response}, attributes, conn) do
+    stripe_response
+    |> CodeCorps.Stripe.Adapters.StripeCustomer.params_from_stripe
+    |> Map.merge(attributes)
```

Comment:
How does this merging work? Are there attributes that are not sent to Stripe or transformed somehow that need merged?